source/reference/operator/aggregation/filter.txt

```diff
@@ -41,10 +41,10 @@ Definition
 
       * - ``as``
 
-        - The variable name for the element in the ``input`` array. The
-          ``as`` expression accesses each element in the ``input``
-          array by this :doc:`variable
-          </reference/aggregation-variables>`.
+        - Optional. The variable name for the element in the ``input`` array. The
+          ``cond`` expression accesses each element in the ``input`` array
+          by this :doc:`variable </reference/aggregation-variables>`.
+          If not specified, ``as`` defaults to ``$$this``.
```

Comment:
Yes I think "this" is a bit clearer. Updated in 1a310637f55e395446de7636072afc7ffb74eb5e